### DISCUSSION TITLE
Welcome to maplibre-navigation-android Discussions!

### DISCUSSION THREAD

**Discussion by boldtrn** [maplibre/maplibre-navigation-android#88] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

**Comment by ianthetechie** [comment]
Thanks for setting this up @boldtrn :)
